src/Administration/Resources/app/administration/src/app/component/form/select/entity/sw-entity-single-select-pagination/sw-entity-single-select-pagination.html.twig

```diff
@@ -0,0 +1,29 @@
+{% block sw_entity_single_select_pagination %}
+    {% block sw_pagination %}
+        <div class="sw-pagination sw-entity-single-select__pagination" v-if="shouldBeVisible">
+            {% block sw_pagination_page_first %}
+                <button :disabled="currentPage === 1" @click.stop="firstPage" class="sw-pagination__page-button">
```

Comment:
maybe you could put the disabled flags into a computed property?

Reply:
Sure. I like my logic out of my templates too :wink: